src/utils.ts

```diff
@@ -347,12 +394,14 @@ export const backportImpl = async (
 
       if (purpose === BackportPurpose.ExecuteBackport) {
         log('backportImpl', LogLevel.INFO, 'Creating Pull Request');
+
+        const comment = await createBackportComment(context, pr);
         const { data: newPr } = await context.github.pulls.create(
           context.repo({
             head: `${tempBranch}`,
             base: targetBranch,
             title: pr.title,
-            body: createBackportComment(pr),
+            body: comment,
```

Comment:
_nit_ the local variable isn't used anywhere else, so why keep it

```suggestion
            body: await createBackportComment(context, pr),
```